Modules/Options/QuestieOptionsDefaults.lua

```diff
@@ -143,7 +143,8 @@ function QuestieOptionsDefaults:Load()
             showPvPQuests = true,
             showAQWarEffortQuests = false,
             showQuestsInNpcTooltip = true,
-            ldbDisplayText = "Questie"
+            ldbDisplayText = "Questie",
+            enableQuestFrameIcons = true
```

Comment:
Checkout the Migration module. We should add another step to enable this new setting for all users on update.